src/main/scala/com/webtrends/harness/component/akkahttp/client/oauth/config/ConfigLike.scala

```diff
@@ -0,0 +1,16 @@
+package com.webtrends.harness.component.akkahttp.client.oauth.config
+
+import java.net.URI
+
+import akka.http.scaladsl.model.HttpMethod
+
+trait ConfigLike {
```

Comment:
Should use akka-http model classes for Uri

Reply:
Good call, converted all URI instances to URI and added some more tests